packages/next/src/client/components/segment-cache-impl/scheduler.ts

```diff
@@ -705,6 +732,21 @@ function diffRouteTreeAgainstCurrent(
             requestTreeChildren[parallelRouteKey] = requestTreeChild
             break
           }
+          case FetchStrategy.PPRDynamic: {
+            // This is a partially dynamic prefetch. Fetch all cacheable data in the tree,
```

Comment:
This is just a naming nit, but I don't think we should let the "dynamic prefetching" name leak into the codebase. Not sure we should call it that publicly once the feature lands. We use "dynamic" to mean the opposite of "cached" in so many places that it invites tons of confusion to use to mean "cached components rendered on demand".

Maybe rename this one to PPR and the other one to StaticPPR?

Reply:
Or CacheComponents and StaticCacheComponents.